cmd/harmony/flags.go

```diff
@@ -955,6 +959,30 @@ var (
 		Usage:    "listen address for pprof",
 		DefValue: defaultConfig.Pprof.ListenAddr,
 	}
+	pprofFolderFlag = cli.StringFlag{
+		Name:     "pprof.folder",
+		Usage:    "folder to put pprof profiles",
+		DefValue: defaultConfig.Pprof.Folder,
+		Hidden:   true,
+	}
+	pprofProfileNamesFlag = cli.StringSliceFlag{
+		Name:     "pprof.profile.names",
+		Usage:    "a list of pprof profile names (separated by ,) e.g. cpu,heap,goroutine",
+		DefValue: defaultConfig.Pprof.ProfileNames,
+		Hidden:   true,
+	}
+	pprofProfileIntervalFlag = cli.IntSliceFlag{
+		Name:     "pprof.profile.intervals",
+		Usage:    "a list of pprof profile interval integer values (separated by ,)",
```

Comment:
The unit is second right? Would you please elaborate a little bit?

Reply:
Yes, the unit is second. Providing for example `--pprof.profile.names cpu,heap --pprof.profile.intervals 0,10` saves the cpu profile on shutdown and the heap profile every 10 seconds. `--pprof.profile.names cpu,heap,goroutine --pprof.profile.intervals 30` saves the cpu, heap and goroutine profile every 30 seconds.

https://github.com/harmony-one/harmony/blob/92568eae89d5cf706b51150ff64e828599d7ed10/cmd/harmony/flags.go#L981-L983